### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by adamdaventryGov**
Worked on the es_indices_ctdc.yml file, single_search_es.yml file, and the crdc-ctdc-private-es.graphql file for the home page.
CTDC-1090

### CODE REVIEW

File: src/main/resources/graphql/crdc-ctdc-private-es.graphql [hunk 1]
```diff
@@ -112,7 +113,22 @@ type StudyDiagnosis {
   ctep_disease_codes: [String]
 }
 
+type HomePage {
+  numberOfParticipants
```
Comment (jonkiky) [inline src/main/resources/graphql/crdc-ctdc-private-es.graphql:117]: does not specific the type of number of participant.

File: src/main/resources/yaml/single_search_es.yml [hunk 2]
```diff
@@ -37,5 +37,15 @@ queries:
...
         - study_short_name
+    result:
+      type: object_array
+  -name: HomePage
+    index:
+      - home_page
+    filter:
+    type: default
+    caseInsensitive: false
+    ignoreIfEmpty:
```
Comment (jonkiky) [inline src/main/resources/yaml/single_search_es.yml:48]: that is not necessary, you can remove the ignoreIfempty this field.

**Review by adamdaventryGov — COMMENTED** [review]
CTDC-1090

**Comment by adamdaventryGov** [comment]
Added properties files and j2 to gitignore and other minor changes

**Review by jonkiky — APPROVED** [review]

**Review by jonkiky — APPROVED** [review]

**Pull request merged**